addons/account/wizard/account_payment_register.py

```diff
@@ -604,6 +608,32 @@ def _compute_hide_writeoff_section(self):
         for wizard in self:
             wizard.hide_writeoff_section = wizard.early_payment_discount_mode
 
+    @api.depends('partner_bank_id', 'amount', 'currency_id', 'payment_method_line_id', 'payment_type', 'communication')
+    def _compute_qr_code(self):
+        for pay in self:
+            qr_html = False
+            if pay.partner_bank_id \
+               and pay.partner_bank_id.allow_out_payment \
+               and pay.payment_method_line_id.code == 'manual' \
+               and pay.payment_type == 'outbound' \
+               and pay.amount \
+               and pay.currency_id:
+                b64_qr = pay.partner_bank_id.build_qr_code_base64(
+                    amount=pay.amount,
+                    free_communication=pay.communication,
+                    structured_communication=pay.communication,
+                    currency=pay.currency_id,
+                    debtor_partner=pay.partner_id,
+                )
+                if b64_qr:
+                    qr_html = '''
+                        <img class="border border-dark rounded" src="{qr_code}"/>
+                        <br/>
+                        <strong>{txt}</strong>
+                        '''.format(txt=_('Scan me with your banking app.'),
+                                   qr_code=b64_qr)
```

Comment:
Could we change this to an f string instead?
```suggestion
                    qr_html = '''
                        <img class="border border-dark rounded" src="{b64_qr}"/>
                        <br/>
                        <strong>{_('Scan me with your banking app.')}</strong>
                    '''
```

Reply:
done !